README.md

```diff
@@ -90,6 +100,7 @@ docker run --rm v2tec/watchtower --help
 ```
 
 * `--host, -h` Docker daemon socket to connect to. Defaults to "unix:///var/run/docker.sock" but can be pointed at a remote Docker host by specifying a TCP endpoint as "tcp://hostname:port". The host value can also be provided by setting the `DOCKER_HOST` environment variable.
+* `--oneshot` Run an update attempt against a container name list one time immediately and exit.
```

Comment:
I think we could use `--interval 0` as a special case for using the `oneshot` mode. Watchtower already has a lot of different flags and some combinations wouldn't make sense, e.g. `--oneshot` and `--interval` therefore we could just use an interval of 0 which indicates no interval.